tests/PSinLinuxCloudShellImage.Tests.ps1

```diff
@@ -11,18 +8,17 @@ Describe "Various programs installed with expected versions" {
         }
     }
 
-    It "Base OS - CBL-D 10" {
+    It "Base OS - CBL-Mariner 2.0" {
 
         [System.Environment]::OSVersion.Platform | Should -Be 'Unix'
         $osDetails = Get-Content /etc/*release
-        $osDetails | Where-Object {$_.Contains('VERSION_ID="10"')} | Should -Not -BeNullOrEmpty
-        $osDetails | Where-Object {$_.Contains('NAME="Common Base Linux Delridge"')} | Should -Not -BeNullOrEmpty
+        $osDetails | Where-Object {$_.Contains('VERSION_ID="2.0"')} | Should -Not -BeNullOrEmpty
+        $osDetails | Where-Object {$_.Contains('NAME="Common Base Linux Mariner"')} | Should -Not -BeNullOrEmpty
```

Comment:
Is this common base linux Mariner or just mariner?

Reply:
It is common base linux mariner (CBL-Mariner)